ocaml-lsp-server/test/e2e/__tests__/completionItem-resolve.test.ts

```diff
@@ -70,12 +72,12 @@ xdescribe("textDocument/completion", () => {
       Types.Position.create(0, 5),
     );
 
-    expect(response).toMatchObject({
-      documentation: {
-        kind: "markdown",
-        value: "`find_all` is another name for `List.filter`.",
-      },
-    });
+    expect(response).toMatchInlineSnapshot(`
+      Object {
+        "documentation": " [find_all] is another name for {!List.filter}. ",
+        "label": "find_all",
+      }
+    `);
   });
 
   it("can get documentation at arbitrary position (before dot)", async () => {
```

Comment:
This should in reality test whether the docs for `LargeFile` are fetched in `Stdlib.LargeFil.in_channel_length`. So the request and response should look like 

```typescript
    let response = await queryCompletionItemResolve(
      "LargeFile",
      Types.Position.create(0, 15),
    );

    expect(response).toMatchInlineSnapshot(`
      Object {
        "documentation": " Operations on large files.
        This sub-module provides 64-bit variants of the channel functions
        that manipulate file positions and file sizes.  By representing
        positions and sizes by 64-bit integers (type [int64]) instead of
        regular integers (type [int]), these alternate functions allow
        operating on files whose sizes are greater than [max_int]. ",
        "label": "LargeFile",
      }
    `);
```

I can commit fix to this PR.

Reply:
That would be great if you could do that :)